scripts/freshvenvs.py

```diff
@@ -8,16 +8,35 @@
 import pathlib
 import sys
 import typing
+from typing import Optional
+
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.resolve()))
 
 from packaging.version import Version
 from pip import _internal
 
+import riotfile
 
-sys.path.append(str(pathlib.Path(__file__).parent.parent.resolve()))
 
 CONTRIB_ROOT = pathlib.Path("ddtrace/contrib")
 
 
+# Venv suite name -> integration name
+module_suite_mapping = {
```

Comment:
It's important to keep the difference between modules and packages clear in this code. I think this would more accurately be called `suite_to_package`, a package being a thing that's downloaded from PyPI.